pages/linux/mons.md

```diff
@@ -0,0 +1,20 @@
+# mons
+
+> A tool to quickly manage two displays.
+> More information: <https://github.com/Ventto/mons>.
+
+- Show primary monitor only:
+
+`mons -o`
+
+- Show second monitory only:
+
+`mons -s`
+
+- Duplicate the primary monitor:
+
+`mons -d`
+
+- Mirror the primary monitor:
```

Comment:
How does this differ from the example above?

Reply:
mirroring preserves the primary's screen resolution, duplication does not

Reply:
Ah, I see. Perhaps we could add this to the description
```suggestion
- Mirror the primary monitor, preserving the resolution:
```